ck.sh

```diff
@@ -77,6 +77,10 @@ case "$COMMAND" in
 				# Replace with new build files.
 				cp -r dev/builder/release/ckeditor/* ../ckeditor/
 
+				(cat ../node_modules/jquery/dist/jquery.min.js && cat ../ckeditor/plugins/a11ychecker/plugin.js) > ../ckeditor/plugins/a11ychecker/plugin.jquery.js
+				mv ../ckeditor/plugins/a11ychecker/plugin.jquery.js ../ckeditor/plugins/a11ychecker/plugin.js
+				rm ../ckeditor/plugins/a11ychecker/plugin.jquery.js
```

Comment:
Unless my eyes are tricking me, I see:

```
cat a b > c
mv c b
rm c
```

I don't think you need `rm c` (and I don't think it will work).